### PR TITLE
Implement calculator

### PR DESCRIPTION
**Pull request by OliviaKRose**

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      let answer = numbers[0];
+
+      let i = 0;
+      numbers.forEach(function (number) {
+        if (i === 0) {
+          i++;
+          return;
+        }
+
+        switch (operator) {
```
Comment (OliviaKRose) [inline calculator.js:25]: Do you currently deal with the case where they want to multiply/use *?

File: calculator.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          case '-':
+            answer = answer - number;
+            break;
+          case '/':
+            answer = answer / number;
+            break;
+          default:
+            console.log("Invalid operator " + operator);
+        }
+        // console.log('index: ' + index);
```
Comment (OliviaKRose) [inline calculator.js:38]: Is this commented out code still required?

File: calculator.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    },
+    calculate: function () {
+      // Return zero if there are no numbers to calculate
+      if (numbers.length === 0) {
+        throw 'Cannot calculate a result because no numbers were given';
+      }
+
+      let answer = numbers[0];
+
+      let i = 0;
```
Comment (OliviaKRose) [inline calculator.js:18]: What is "i" used for?

File: index.js [hunk 4]
```diff
@@ -1,7 +1,20 @@
...
+const c = parseInt(readline.prompt());
+
+const calculator = calc.create_calculator(operator);
+
+for (var i = 0; i < c; i++) {
+  console.log('Enter a number:');
+  const n = parseInt(readline.prompt());
+  calculator.add_number(n);
+}
+
```
Comment (OliviaKRose) [inline index.js:19]: Do we have any error handling that covers if they input something that is not a number?

File: test/calculator_test.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      calculator.add_number(1486);
+      calculator.add_number(-402);
+      calculator.add_number(4);
+
+      assert.equal(1077, calculator.calculate());
+    });
+
+    it('rejects the calculation when there are no numbers', function () {
+      const calculator = calc.create_calculator('+');
+
```
Comment (OliviaKRose) [inline test/calculator_test.js:40]: These tests look good! Would you be able to add some more to cover the other operators? Also edge cases i.e. trying to divide by 0 or non-number inputs?
